### PR TITLE
Add 'Table of Contents' to articles

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
if article has > 1 subheaders

e.g.

toc

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+
+@import "components/article/main";
+@import "components/more-on/main";
+@import "components/timeline/main";
 @import "components/capi2-related/main";
-$o-quote-is-silent: false;
-@import "o-quote/main";
-$o-big-number-is-silent: false;
-@import "o-big-number/main";
-@import "o-ft-icons/main";
```
Comment (ironsidevsquincy) [inline client/main.scss]: changed import order to allow overriding of origami styles

File: gulpfile.js [hunk 2]
```diff
@@ -62,7 +62,8 @@ gulp.task('sourcemap', ['minify-js'], function(){
 });
 
 gulp.task('watch', function() {
-	gulp.watch('./client/**/*', ['default']);
+	gulp.watch('./client/**/*.js', ['build-js']);
+	gulp.watch('./client/**/*.scss', ['build-sass']);
```
Comment (ironsidevsquincy) [inline gulpfile.js]: only build type that changed

File: views/layout.html [hunk 3]
```diff
@@ -11,27 +11,35 @@ <h3 class="article__title">{{{article.titleHTML}}}</h3>
...
+		</div>
+		{{#if showTOC}}
+			<div data-o-grid-colspan="12 XLoffset2">
+				<div class="article__toc o-ft-forms__field-group">
+					<select class="article__toc__items o-ft-forms__field--select">
+						<option disabled selected>Table of contents</option>
+						{{#each subheaders}}
+							<option value="{{this.id}}">{{this.text}}</option>
+						{{/each}}
+					</select>
```
Comment (ironsidevsquincy) [inline views/layout.html]: also, move the meta out of the LHC

Comment (commuterjoy) [inline views/layout.html]: can you add `data-trackable="toc"` and check it sends a beacon when clicked

**Comment by ironsidevsquincy** [comment]
happy for this to merge @matthew-andrews?

**Comment by matthew-andrews** [comment]
:shipit:

**Pull request merged**